srcpkgs/swaync/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'swaync'
+pkgname=swaync
+version=0.4
+revision=1
+wrksrc=SwayNotificationCenter-${version}
+build_style=meson
+hostmakedepends="vala pkg-config"
```

Comment:
```suggestion
hostmakedepends="vala glib-devel pkg-config"
```
This should fix cross builds if there isn't anything else hidden below the current error.